content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -98,6 +98,63 @@ helm upgrade --install operator wandb/operator -n wandb-cr --create-namespace
 ```
 3. Configure the W&B operator custom resource to trigger the W&B Server installation. Create an operator.yaml file to customize the W&B Operator deployment, specifying your custom configuration. See [Configuration Reference]({{< relref "#configuration-reference-for-wb-operator" >}}) for details.
```

Comment:
```suggestion
3. Configure the W&B operator custom resource to trigger the W&B Server installation. Copy this example configuration to a file named `operator.yaml`, so that you can customioze your W&B deployment. Refer to [Configuration Reference]({{< relref "#configuration-reference-for-wb-operator" >}}).
```